components/coding/GoalsSectionComponent.tsx

```diff
@@ -7,6 +7,7 @@ import ExpandableContainer from "./ExpandableContainer";
 export type GoalsSectionProps = {
   sectionName?: string;
   userGoals?: UserGoalsData[];
+  profile?: boolean;
```

Comment:
Can you explain how we're using this new type? I think I get what you're doing here - if we're calling this component on the profile, set to true, set to false if not. But I'm not sure I see where it's actually being used?

nitpick on naming - I'd probably change the name so it's more apparent it's a boolean. Maybe like onProfile or inProfile

Reply:
I'm only using the new type here because we are using the GoalsSectionComponent right now for the goals index page and on the profile index page v2.  This is a problem because if there are no userGoals for the user on the profile page, I want to display the box saying there are no user goals.  If there are no userGoals in a section on the goals index page, like "archived" or "completed" or "current" then the same box will display and that looks bad I think. If the user has no goals it's better to see this I think: 
![Screen Shot 2022-12-19 at 5 46 06 PM](https://user-images.githubusercontent.com/111075855/208549562-d3fe89c1-ef9f-49c2-b3c8-c7208a07bea5.png)

Maybe this would be better?

Reply:
I think either the inProfile or enum approach both work - I'd probably lean towards the inProfile boolean because it's simpler and I don't anticipate us placing this in too many other places? If we needed to place it on more than two different pages and specify different behavior for each page, might make sense to use an enum

Reply:
changed to inProfile.  What you're saying makes sense.